spring-integration-redis/src/test/java/org/springframework/integration/redis/config/RedisInboundChannelAdapterParserTests.java

```diff
@@ -76,6 +78,16 @@ public void validateConfiguration() {
 		assertEquals(converterBean, accessor.getPropertyValue("messageConverter"));
 		assertEquals(context.getBean("serializer"), accessor.getPropertyValue("serializer"));
 
+		Object container = accessor.getPropertyValue("container");
+		DirectFieldAccessor containerAccessor = new DirectFieldAccessor(container);
+		Object taskExecutorObj = containerAccessor.getPropertyValue("taskExecutor");
+		assertEquals(ThreadPoolTaskExecutor.class, taskExecutorObj.getClass());
+		ThreadPoolTaskExecutor taskExecutor = (ThreadPoolTaskExecutor) taskExecutorObj;
+		assertEquals(5, taskExecutor.getCorePoolSize());
+		assertEquals(10, taskExecutor.getMaxPoolSize());
+		DirectFieldAccessor executorAccessor = new DirectFieldAccessor(taskExecutor);
+		assertEquals(25, executorAccessor.getPropertyValue("queueCapacity"));
```

Comment:
Well, we don't need to assert exector's properties. There is just enough to inject that `executor` and perform `assertSame()` against `containerAccessor.getPropertyValue("taskExecutor")`